src/renderer/components/NewProjectModal/RunTranscriptionView.tsx

```diff
@@ -70,6 +70,11 @@ const RunTranscriptionView = ({ closeModal, nextView }: Props) => {
     ) {
       return;
     }
+
+    if (currentProject.filePath === null) {
```

Comment:
I don't think 'filePath' is an attribute in 'project' any more - we have 'mediaFilePath' and 'saveFilePath'

Reply:
refactor done here: https://github.com/patrickbrett/mlvet/pull/38/commits/5d57bfca861469a5898825e6172929ba38319e0c